gremlin-go/driver/traversal.go

```diff
@@ -297,11 +297,11 @@ func newP(operator string, args ...interface{}) Predicate {
 }
 
 func newPWithP(operator string, pp p, args ...interface{}) Predicate {
-	values := make([]interface{}, len(args)+1)
+	values := make([]interface{}, 1)
+	values[0] = pp
 	for _, arg := range args {
 		values = append(values, arg)
 	}
```

Comment:
```suggestion
	values = append(values, args...)
```

Reply:
I thought these were once changed? I'll change them in my IDE and push. Committing suggestion might be too messy.